src/components/CommentsList.jsx

```diff
@@ -0,0 +1,31 @@
+import { useContext, useState, useEffect } from "react";
+import { CommentsContext } from "../contexts/CommentsContext";
+import { getComments } from "../api";
+import { Comment } from "./Comment";
+
+export const CommentsList = ({ article_id }) => {
+  console.log();
+  const { comments, setComments } = useContext(CommentsContext);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getComments(article_id)
+      .then(({ comments }) => {
+        setComments(comments);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
Later on you may want to add an error state and conditionally render some user feedback